### DISCUSSION TITLE
Capture "real time" position for many CFs

### DISCUSSION THREAD

**Discussion by Dhipz** [USC-ACTLab/crazyswarm#662] [Q&A] [answered]
Hi, so currently i'm able to fly up to 8 CFs simultaneous using Lighthouse V2 and 1 Crazyradio without using `enable_logging` and `enable_logging_pose`. But when i try to enable logging i can only fly up to 3 CFs when capturing their positions using `position()` function. Is there a way to capture many CFs (6 or more to be exact) positions at the same time (or a better way to capture it)? I only need the x and y axis position and it only be used for logging so it doesn't need to be fully real time (a delay of 0.2-0.5 sec between capture is fine), if it helps.

Thank you in advance!

**Comment by jpreiss** [comment, marked as answer]
`enable_logging_pose` uses a hard-coded frequency of 10Hz:

https://github.com/USC-ACTLab/crazyswarm/blob/ea4a67e57743fb7c8f56270edd96b191876f4057/ros_ws/src/crazyswarm/src/crazyswarm_server.cpp#L609

You can edit this and rerun `build.sh` to change it.

However, 10Hz logging does not use that much bandwidth. I am surprised it didn't work with 6 CFs. What control modes are you using? Are you logging anything else?

**Reply by Dhipz** [reply]
Hi thanks for your reply!

Well when i tried logging (by logging i mean flying and calling `cf.position()` every now and then) for 6 CFs i only got at most 4 (using 2 radios) of their position while the other 2 looks like didn't send their position and the whole program just stuck waiting for those 2 position. I think i left most of the settings as default so the control modes should be mellinger, maybe i should have tried using PID and test it. Anyway i ended up using only 1 CF for my project so i'll mark your reply as the solution.

Thank you
